packages/e2e-tests/helpers/utils.ts

```diff
@@ -84,6 +84,24 @@ export async function mapLocators<T>(
   );
 }
 
+export async function getSupportFormErrorDetails(page: Page) {
+  if (await page.locator('[data-test-id="SupportForm"]').isVisible()) {
+    const errorDetailsLocator = page.locator('[data-test-id="UnexpectedErrorDetails"]');
+    return (await errorDetailsLocator.innerText()) || "(support form is visible)";
```

Comment:
This will throw an error (instead of returning "(support form is visible)") if the `SupportForm` has no error details. You probably want something like `return (await errorDetailsLocator.isVisible()) ? await errorDetailsLocator.innerText() : "(support form is visible";`

Reply:
Ah! Learned something new.
How is this?

Reply:
Many locator methods work that way: they wait for the located element to become visible and throw a `TimeoutError` if that doesn't happen in time. `locator.isVisible()` is one of those that doesn't wait (and doesn't throw).

Reply:
Ah whoops... What I meant was: "How is this new version?" - But thanks, it confirms my guesses 😆